azure-pipelines.yml

```diff
@@ -408,10 +408,18 @@ jobs:
     parameters:
       downloadPartialBuildBinFrom:
         - cw310_sw_build
+  - bash: |
+      ci/scripts/get-bitstream-strategy.sh '^sw\/' 'testplan\.hjson$' '^site\/' '^doc\/' '^COMMITTERS' '^CLA' '\.md$'
```

Comment:
nit: I just took a very quick look but wanted to make sure that we will splice even if we use the cached bitstream, right (to handle changes in rom and test_rom ?

Reply:
That's correct. This job's output is a bitstream containing the test_rom that was built previously in this pipeline. The prepare-cached-bitstream.sh script does the splicing for the cached bitstream w/ newly-built test_rom.

There's a later job that splices the rom, and that is none-the-wiser about whether the bitstream was built or cached. It always runs against whatever this job provides it.